p2p/connected_test.go

```diff
@@ -18,24 +19,27 @@ func TestAnnounceConnected(t *testing.T) {
 		return
 	}
 
-	if err := p2ptest.ConnectQriPeers(ctx, testNodes); err != nil {
-		t.Error(err.Error())
-		return
-	}
-
 	// Convert from test nodes to non-test nodes.
 	nodes := make([]*QriNode, len(testNodes))
 	for i, node := range testNodes {
 		nodes[i] = node.(*QriNode)
 	}
 
+	for i, a := range nodes {
```

Comment:
We shouldn't be changing this, the semantics between ConnectQriPeers and Host.Connect are completely separate. See my other comments in p2p/peers.go